src/Microsoft.Diagnostics.ExtensionCommands/Host/CommandExtensions.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.Diagnostics.DebugServices;
+using Microsoft.Diagnostics.ExtensionCommands.Output;
+using Microsoft.Diagnostics.Runtime;
+
+namespace Microsoft.Diagnostics.ExtensionCommands
+{
+    public static class CommandExtensions
+    {
+        public const string DacTableSymbol = "g_dacTable";
+        public const string DebugHeaderSymbol = "DotNetRuntimeDebugHeader";
+
+        /// <summary>
+        /// Displays the special diagnostics info header memory block (.NET Core 8 or later on Linux/MacOS)
+        /// </summary>
+        public static void DisplaySpecialInfo(this CommandBase command, string indent = "")
+        {
+            if (command.Services.GetService<ITarget>().OperatingSystem != OSPlatform.Windows)
+            {
+                ulong address = SpecialDiagInfoHeader.GetAddress(command.Services);
```

Comment:
Most of these special info aspects feel like they should be either on the runtime they relate to or the target.

Reply:
Like I said above I look at them as command base extensions.  They print things out.  They need to use the same set of services the command calling them is using.

Reply:
SpecialDiagInfoHeader is a global struct at special address in memory add by createdump. It isn't associated with a runtime. We can get the runtime module address out of it which is why it calls the other runtime module specific ones to display that stuff.